README.md

```diff
@@ -16,6 +16,14 @@ $ hexo server
 
 Translation for this documentation project are currently maintained in separate repositories forked from this original one.
 
+### Spanish
+
+Spanish translation is maintained by CO-Vuejs.
+
+* Translation Repo — [/co-vuejs/es.vuejs.org](https://github.com/co-vuejs/es.vuejs.org)
+* Primary maintainers - [CamiloRincon](https://github.com/CamiloRincon), [adrianengine](https://github.com/adrianengine), [diegofelipemoreno](https://github.com/diegofelipemoreno), [yamitvillamil](https://github.com/yamitvillamil)
```

Comment:
Following the example of other translations, I think is better to show just one Primary maintainer.
In this case: [Camilo Rincon](https://github.com/CamiloRincon)